norona_marynicole/countries.js

```diff
@@ -0,0 +1,95 @@
+function searchCountry() {
+  let countryName = document.getElementById("countryInput").value.trim();
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
+      "<p>Please enter a country name.</p>";
+    document.getElementById("sameRegionCountries").innerHTML = "";
+    document.getElementById("newCountryDetails").innerHTML = "";
```

Comment:
IDs should be written in snake case. All lower case separated by underscore

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5